balancer/weightedtarget/weightedaggregator/aggregator.go

```diff
@@ -222,6 +236,14 @@ func (wbsa *Aggregator) clearStates() {
 func (wbsa *Aggregator) BuildAndUpdate() {
```

Comment:
I believe we can unexport this now?

Reply:
sure. I believe we can remove this and keep just the Locked version of `buildAndUpdate`. making that update in the rev